sdk/python/lib/test/automation/test_local_workspace.py

```diff
@@ -120,23 +120,23 @@ def test_stack_settings(self):
 
     def test_plugin_functions(self):
         ws = LocalWorkspace()
-        # Install aws 3.0.0 plugin
-        ws.install_plugin("aws", "v3.0.0")
+        # Install aws 6.1.0 plugin
+        ws.install_plugin("aws", "v6.1.0")
         # Check the plugin is present
         plugin_list = ws.list_plugins()
-        self.assertTrue(found_plugin(plugin_list, "aws", "3.0.0"))
+        self.assertTrue(found_plugin(plugin_list, "aws", "6.1.0"))
 
         # Remove the plugin
-        ws.remove_plugin("aws", "3.0.0")
+        ws.remove_plugin("aws", "6.1.0")
         # Check that the plugin has been removed
         plugin_list = ws.list_plugins()
-        self.assertFalse(found_plugin(plugin_list, "aws", "3.0.0"))
+        self.assertFalse(found_plugin(plugin_list, "aws", "6.1.0"))
 
     def test_stack_functions(self):
         project_settings = ProjectSettings(name="python_test", runtime="python")
         ws = LocalWorkspace(project_settings=project_settings)
-        stack_1_name = f"python_int_test_first_{get_test_suffix()}"
-        stack_2_name = f"python_int_test_second_{get_test_suffix()}"
+        stack_1_name = get_test_org() + "/" + f"first_{get_test_suffix()}"
```

Comment:
I wanted to use fully qualified stack names for this (with the `python_test` project), but for some reason the project name wasn't in the names when the stacks were listed?